src/loaders/compute_tools/mash/mash.py

```diff
@@ -25,6 +26,7 @@ def _run_mash_single(
                                    # Save result file to source file directory. The suffix '.msh' will be appended.
                '-k', f'{kmer_size}',
                '-s', f'{sketch_size}',
+               '-p', program_threads,
```

Comment:
I didn't find anywhere online the default for mash.

Reply:
```
~/kbase/mash/mash-Linux64-v2.3$ ./mash sketch -h

Version: 2.3

Usage:

  mash sketch [options] <input> [<input>] ...

Description:

  Create a sketch file, which is a reduced representation of a sequence or set
  of sequences (based on min-hashes) that can be used for fast distance
  estimations. Inputs can be fasta or fastq files (gzipped or not), and "-" can
  be given to read from standard input. Input files can also be files of file
  names (see -l). For output, one sketch file will be generated, but it can have
  multiple sketches within it, divided by sequences or files (see -i). By
  default, the output file name will be the first input file with a '.msh'
  extension, or 'stdin.msh' if standard input is used (see -o).

Options:

  Option     Description (range) [default]

  -h         Help

  -p <int>   Parallelism. This many threads will be spawned for processing. [1]```

Reply:
Was just worried we might be screwing things up by allowing ppl to set low threads, but this seems ok

Reply:
:+1: